podaac/subsetter/subset.py

```diff
@@ -1021,6 +1022,7 @@ def subset(file_to_subset, bbox, output_file, variables=None,  # pylint: disable
     # If dataset has groups, transform to work with xarray
     if has_groups:
         nc_dataset = transform_grouped_dataset(nc_dataset, file_to_subset)
+        nc_dataset = dc.remove_duplicate_dims(nc_dataset)
```

Comment:
Do we only need to remove duplicate dimensions on grouped datasets? I would think it would be on any dataset